README.md

```diff
@@ -1,7 +1,9 @@
 # nghttp2-sys
 
 A common library for linking `nghttp2` to rust programs (also known as
-libnghttp2).
+libnghttp2).  By default, it uses a bundled copy of `libnghttp2`.  If that
+is not desired, you can use the system's `libnghttp2` instead by not enabling
+the default `vendored` feature.
```

Comment:
It is conventional to support an environment variable such as `LIBNGHTTP2_SYS_USE_PKG_CONFIG=1` so that people who aren't the maintainers of an application can easily toggle vendored dependencies to use the system copy.